roscala/src/main/scala/coop/rchain/rosette/prim/Prim.scala

```diff
@@ -8,7 +8,9 @@ import scala.reflect.{classTag, ClassTag}
 sealed trait PrimError
 case class ArgumentMismatch(msg: String)               extends PrimError
 case class TypeMismatch(argNum: Int, typeName: String) extends PrimError
-case object ArithmeticError                            extends PrimError
+case object ArithmeticError extends PrimError
```

Comment:
The formatting seems off. Did you run scalafmt (should be run when you compile).

Reply:
Sorry @tymm , it seems that format task didn't run when i compiled the roscala project. So, i ran sbt scalafmt task manually for roscala project.